### DISCUSSION TITLE
Example with SVGR

### DISCUSSION THREAD

**Discussion by jeroenroumen** [RyanClementsHax/storybook-addon-next#56] [Q&A] [answered]
So I was setting up a blank Next 12 project with this plugin. For images it works pretty good without configuring anything. But for our project we depend on SVGR to import SVGs as React components. When trying to load a component through Storybook with this plugin I get a weird error message.

<img width="1166" alt="image" src="https://user-images.githubusercontent.com/17384485/159995229-a617e326-8e12-4f76-8354-5eb191b673f1.png">

When console logging the import I notice that when loaded through Storybook, it still loads the SVG with the `next-image-loader` because it gives me an object of type `StaticImageData` and not a React component/function. Any idea on how to set this up without modifying the Storybook main.js?

In the `next.config.js` I've set it up like so: https://react-svgr.com/docs/next/ and it works fine in the Next application itself.

**Comment by RyanClementsHax** [comment, marked as answer]
This is being addressed in #57
